container/container_unix.go

```diff
@@ -343,6 +343,17 @@ func (container *Container) BuildCreateEndpointOptions(n libnetwork.Network) ([]
 		createOptions = append(createOptions, libnetwork.EndpointOptionGeneric(genericOption))
 	}
 
+	/* FIXME: When restoring, the container should get the same IP address it had before, so
```

Comment:
@boucher , yes, you're technically correct about IPs -- upon restore criu will go ahead and restore the net namespace in its original state. This can be work-around-ed by editing the images, but let's better discuss what's the network c/r mode Docker needs.